cookbooks/bcpc/attributes/default.rb

```diff
@@ -104,6 +104,7 @@
 default['bcpc']['bootstrap']['vip'] = node['bcpc']['bootstrap']['server']
 default['bcpc']['bootstrap']['dhcp_range'] = '10.0.100.14 10.0.100.250'
 default['bcpc']['bootstrap']['dhcp_subnet'] = '10.0.100.0'
+default['bcpc']['bootstrap']['search_domain'] = 'bcpc.example.com'
```

Comment:
We already have domain; why duplicate this or can it be set to the previous node attribute by default?